gce/gce.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package gce provides methods for querying the gce metadata
+package gce
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"golang.org/x/crypto/ssh"
+)
+
+// GetAuthorizedKeys retrieves the authorized ssh keys from the metadata server running on a GCE instance
```

Comment:
We tend to wrap comment lines at 80 columns.  You could probably just shorten this up (it's in the metadata package, so we know where the data is coming from)

// GetAuthorizedKeys returns the set of authorized SSH keys.

Reply:
done